### PR TITLE
Add Polish tasks (PL-MTEB)

### PR DESCRIPTION
**Pull request by rafalposwiata**
Add Polish tasks (PL-MTEB). 
Files structure was inspired by Chinese C-MTEB. 
Evaluation results for the three models have already been added to https://huggingface.co/datasets/mteb/results

### CODE REVIEW

**Comment by rafalposwiata** [comment]
> * Do you know of any Polish Reranking tasks? I think we should be able to find something for MMarcoReranking at least (which is also there for C-MTEB), as it's available in multiple languages I think?

I searched but couldn't find any Polish Reranking tasks. Unfortunately, mMarco doesn't include Polish. 

> * Would it be okay for you if I add a new Polish tab in the Overall tab of the [leaderboard](https://huggingface.co/spaces/mteb/leaderboard) that aggregates across these tasks added by you as well as https://github.com/embeddings-benchmark/mteb/pull/121 for Retrieval similar to the Chinese tab?

Of course, I will be grateful :)

**Review by Muennighoff — COMMENTED** [review]
Looks good to me! I've added the tab to the leaderboard: https://huggingface.co/spaces/mteb/leaderboard

Two more things before mering
- Can you add each dataset to the README.md table? You can consult this script if you want to: https://github.com/embeddings-benchmark/mteb/blob/main/scripts/data/create_task_table.py
- Can you maybe a few more models? I think running [multilingual-e5-large](https://huggingface.co/intfloat/multilingual-e5-large) would be great
- Can you run BEIR-PL Retrieval for the models too, so we have the average score? You will have to checkout to [this branch](https://github.com/embeddings-benchmark/mteb/pull/121)

Once done, will merge this & BEIR-PL. Really amazing work!!!

**Comment by Muennighoff** [comment]
Nice the code is good now - Merging this! Let me know if you manage to run the additional models/evals!

**Review by Muennighoff — APPROVED** [review]

File: mteb/tasks/STS/PolishSTS.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            "description": "Compositional Distributional Semantics Corpus for textual relatedness.",
+            "reference": "https://aclanthology.org/P17-1073.pdf",
+            "type": "STS",
+            "category": "s2s",
+            "eval_splits": ["test"],
+            "eval_langs": ["pl"],
+            "main_score": "cosine_spearman",
+            "min_score": 1,
+            "max_score": 5
+        }
```
Comment (Muennighoff) [inline mteb/tasks/STS/PolishSTS.py:37]:
```suggestion
        }
        
```

**Pull request merged**